MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -127,6 +134,11 @@ protected internal override void Initialize(IntermediateSerializer serializer)
             }
         }
 
+        public override bool CanDeserializeIntoExistingObject
+        {
+            get { return TargetType.IsClass; }
```

Comment:
I think this actually is more complex...  like I suspect `Array` types can't support it.  Don't know if we have a test that covers that case.